printing/printer.ml

```diff
@@ -549,10 +549,14 @@ let pr_evgl_sign (type a) env sigma (evi : a evar_info) =
   in
```

Comment:
This printer and the one in termops look similar, are they actually different?

Reply:
They don't use the same base printer for constrs, so I'd say yes (although it might be possible to factor this out somehow).

Reply:
more importantly AFAICT only the termops one prints evar bodies

Reply:
so we can do this https://github.com/coq/coq/pull/17227